src/test/java/core/basesyntax/service/impl/FileReaderServiceImplTest.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.exceptions.FruitTransactionException;
+import core.basesyntax.service.FileReaderService;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class FileReaderServiceImplTest {
+    private static FileReaderService readerService;
+
+    @BeforeAll
+    static void beforeAll() {
+        readerService = new FileReaderServiceImpl();
+    }
+
+    @Test
+    public void test_readFromFile_ok() {
```

Comment:
It's a test for sure as it has `@Test`, you can remove it from the test method names.